azure/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -1,6 +1,6 @@
 # Self-hosted Kubernetes assets (kubeconfig, manifests)
 module "bootkube" {
-  source = "github.com/kinvolk/terraform-render-bootkube?ref=aec4326328bb7bdbd132a6e0b9c202e40f1ec04c"
+  source = "github.com/kinvolk/terraform-render-bootkube?ref=b11e5ce6dd86b064b9b0611fbc0e2776160d0d20"
```

Comment:
nit pick, but shouldn't this be the merge commit now that is merged? I mean 7e9cf2460a3f01703f2edbfde463c1843fbf8d49

Reply:
done!